wls-gui-wahllokalsystem/src/plugins/router.ts

```diff
@@ -11,7 +19,21 @@ const routes = [
     component: HomeView,
     meta: {},
   },
-  { path: "/:catchAll(.*)*", redirect: "/" }, // CatchAll route
+  {
+    path: "/newroute",
+    name: EXAMPLE_ROUTES_NEWROUTE,
+    component: ExampleNewRouteView,
```

Comment:
Das der Name aus einer anderen Quelle kommt als die Component finde ich nicht so gut. Du könntest `ExampleNewRouteView.__name` verwenden. Ebenso auch bei den `to`-Attributen.